src/overlay/FlowControlCapacity.cpp

```diff
@@ -205,10 +210,19 @@ FlowControlCapacity::hasOutboundCapacity(StellarMessage const& msg) const
 }
 
 uint64_t
-FlowControlCapacity::msgBodySize(StellarMessage const& msg)
+FlowControlCapacity::msgBodySize(StellarMessage const& msg,
+                                 uint32_t remoteVersion, uint32_t localVersion)
 {
-    return static_cast<uint64_t>(xdr::xdr_size(msg) -
-                                 xdr::xdr_size(msg.type()));
+    if (remoteVersion >= Peer::FIRST_VERSION_UPDATED_FLOW_CONTROL_ACCOUNTING &&
```

Comment:
Nit: Brief comment here like "calculation changed from size of full stellar message to size of tx envelope only". This XDR math is not particularly easy to follow on its own.

Reply:
Agreed, added a comment.